packages/paste-website/src/pages/foundations/content/product-style-guide/index.mdx

```diff
@@ -40,482 +40,175 @@ export const getStaticProps = async () => {
+### Parentheses
 
-**Do** use numerals (1, 2, 3, 45, 101) instead of spelling out the number.
+Use parentheses to provide an example, explanation, or to note an [acronym](#acronyms). Do not use parentheses for whole sentences.
 
-- Percentages _49%_
+### Periods
 
-**Do** write expressions that use ordinal numbers in words.
+End every full sentence with a period.
 
-- _One to watch, thousands of options_
-- _First, create a profile_
+Don't use periods at the end of fragments or incomplete sentences.
 
-**Do** use both words and numerals to represent numbers over 1000, unless the user needs a specific number.
+Don't use periods at the end of an anchor link if it is the last line of a piece of content.
 
-- _Up to 40k emails_
-- _More than $1.5B_
-- **Note** this is most commonly used in marketing content — make sure you're giving the user what they need to complete a task, and not overloading them with extra info.
+### Quotation marks
 
-**Do** use a space between the number and its unit of measurement or its order of magnitude if this is written out in words.
-
-- _40 GB, 564 customers_
-- _More than $1.3 billion, 45 million customers_
-
-**Do** use commas in numbers that are four or more digits.
-
-- _1,365 messages sent_
-
-**Don't** use commas in error codes, account numbers, or phone numbers.
-
----
-
-#### Sentence case
-
-**Do** use sentence case everywhere, even in H1s, CTAs, and navigation. Sentence case makes content feel more human and it's generally easier to scan.
-
-- This is sentence case. (Use this.)
-- This is Title Case. (Rarely use this, and only if directed.)
-- THIS IS ALL CAPS. (Don’t use this.)
-
----
-
-#### Time
-
-**Do** write times so that they are easy to scan and sound human to read aloud. Usually, this means using the 12hr clock, with am/pm modifiers.
-
-- _12:25am_
-- _3:32pm_
-
----
-
-## Common errors
-
-#### e.g./i.e.
-
-- _e.g._ = for example
-  - _Create a memorable name for your Flex Project, e.g. "Flex Training 2020."_
-- _i.e._ = that is
-  - Tip: _i.e._ usually precedes a simpler way of saying something. Say it simply the first time. Avoid writing _i.e._ if you can.
-
----
-
-#### Every day/everyday
-
-- _Every day_ = each day
-  - _Twilio supports thousands of customers every day._
-- _Everyday_ = ordinary/usual
-  - _Twilio supports everyday customer needs, like call routing._
-
----
-
-#### It's/its
-
-- _It's_ = contraction of "it"+"is"
-  - _It's now possible to view real-time queue stats._
-  - Tip: Can you say _it is_ instead of the word you want? Does the sentence make sense? You want _it's_.
-- _Its_ = possessive pronoun (belonging to it)
-  - _The company and its customers._
-
----
-
-#### Log in/login
-
-- _Log in_ = verb
-  - _To access your account, please log in._
-- _Login_ = noun/adjective
-  - _Enter your login details._
-
----
-
-#### Set up/setup
-
-- _Set up_ = verb
-  - _Set up your profile._
-- _Setup_ = noun/adjective
-  - _System setup information can be found in the user guide._
```

Comment:
Can you add "set up/setup" to the word list?

Reply:
word list is to define which words to use when or when to never use words (e.g. we say "select" not "choose", and avoid the word "easy" or "easily").

set up / setup and sign up / sign-up is, sadly, basic grammar. people should check Twilio's preferred dictionary (Miriam Webster). I'd really love for us to have a tool that embedded in Figma (like Writer) that auto-checked all this but alas, here we are.

Reply:
truuuuue, fair enough